### PR TITLE
Homekit troubleshooting section

### PR DESCRIPTION
**Pull request by cdce8p**
**Description:**
Due to many new issues being opened about problems with the `HomeKit` component, I've added a `troubleshooting` section to the documentation.

I've included some small improvement to the `setup` section as well.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/homekit.markdown [hunk 1]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
-2. Reproduce the bug / problem you have encountered.
-3. Stop Home Assistant and copy the log from the log file. That is necessary since some errors only get logged, when Home Assistant is being shutdown.
-4. Follow this link: [home-assistant/issues/new](https://github.com/home-assistant/home-assistant/issues/new?labels=component: homekit) and open a new issue.
-5. Fill out all fields and especially include the following information:
-   - The configuration entries for `homekit` and the `component` that is causing the issue.
-   - The log / traceback you have generated before.
-   - Screenshots of the failing entity in the `states` panel.
 
-## {% linkable_title Troubleshooting PIN not appearing %}
+**PIN doesn't appear as persistent status**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 2]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+You might have paired the `Home Assistant Bridge` already. If not, delete the `.homekit.state` file ([guide](#deleting-the-homekitstate-file)).
 
-In some instances, the PIN will not appear as a persistent status or in the log files despite deleting `.homekit.state`, enabling logging, and reboot.  The log files will include the error ```Duplicate AID found when attempting to add accessory```.
+**`Home Assistant Bridge` doesn't appear in the Home App (for pairing)**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 3]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+For `Docker` users: make sure to set `network_mode: host`. Other reasons could be network related. Make sure to check your router configuration. For some it helped when the Home Assistant device was using WIFI, not LAN. Remember that the iOS device needs to be in the same local network as the Home Assistant device for paring.
 
-In such cases, modifying your configuration.yaml to add a filter limiting the included entities similar to the following:
+**Pairing hangs - zeroconf error**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 4]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
-  include_domains:
-    - light
-```
+**Duplicate AID found when attempting to add accessory**<br>
+Two of your entities share the same `entity_id`. Either resolve this or configure the [filter](#configure-filter) to exclude them.
+
+
+### {% linkable_title Issues during normal use %}
+
+**Pairing hangs - no error**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 5]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
-In such cases, modifying your configuration.yaml to add a filter limiting the included entities similar to the following:
+**Pairing hangs - zeroconf error**<br>
+Paining eventually fails, you might see and an error message `NonUniqueNameException`. To resolve this, you need to replace a specific file. See the following git issues for more details: [home-assistant#14567](https://github.com/home-assistant/home-assistant/issues/14567) and [home-assistant#17181](https://github.com/home-assistant/home-assistant/issues/17181)
 
-```yaml
-filter:
-  include_domains:
-    - light
-```
+**Duplicate AID found when attempting to add accessory**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 6]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+Two of your entities share the same `entity_id`. Either resolve this or configure the [filter](#configure-filter) to exclude them.
+
+
+### {% linkable_title Issues during normal use %}
+
+**Pairing hangs - no error**<br>
+Make sure that you don't try to add more then 100 accessories, see [device limit](#device-limit). In rare cases one of your entities doesn't work with the HomeKit component. Use the [filter](#configure-filter) to find out which one. Feel free to open a new issue in the `home-assistant` repo, so we can resolve it.
+
+**Some of my devices don't show up - Z-Wave / Discovery**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 7]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+See [disable auto start](#disable-auto-start)
+
+**Accessory not responding - after restart or update**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 8]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+See [device limit](#device-limit)
+
+**Accessory not responding - randomly**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

File: source/_components/homekit.markdown [hunk 9]
```diff
@@ -280,36 +273,62 @@ The following components are currently supported:
...
+Unfortunately that sometimes happens at the moment. It might help to close the `Home` App and delete it from the cache. Usually the accessory should get back to responding after a few minutes at most.
 
-Restart Home Assistant and re-attempt pairing - a persistent status should now correctly appear.
+**Accessories not responding / behaving unusual - Upgrade from `0.65.x`**<br>
```
Comment (DubhAd) [inline source/_components/homekit.markdown]: Should be linkeable title

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Comment by cdce8p** [comment]
@DubhAd Personally I think that it looked better inline (with strong and line break), then as a title, due to the bottom padding. However I agree that it might make sense to link to one specifically.

**Comment by DubhAd** [comment]
It makes a huge difference when you're directing people to the relevant troubleshooting section, rather than just a header and then trying to describe which section to use.

**Review by DubhAd — APPROVED** [review]

**Pull request merged**